src/Servers/Kestrel/Transport.Quic/src/Internal/QuicLog.cs

```diff
@@ -203,6 +204,12 @@ public static void StreamReused(ILogger logger, QuicStreamContext streamContext)
     [LoggerMessage(19, LogLevel.Warning, $"{nameof(SslServerAuthenticationOptions)} must provide at least one application protocol using {nameof(SslServerAuthenticationOptions.ApplicationProtocols)}.", EventName = "ConnectionListenerApplicationProtocolsNotSpecified")]
     public static partial void ConnectionListenerApplicationProtocolsNotSpecified(ILogger logger);
 
+    [LoggerMessage(20, LogLevel.Debug, "QUIC listener starting with configured endpoint {listenEndPoint}.", EventName = "ConnectionListenerStarting")]
+    public static partial void ConnectionListenerStarting(ILogger logger, IPEndPoint listenEndPoint);
+
+    [LoggerMessage(22, LogLevel.Debug, "QUIC listener aborted.", EventName = "ConnectionListenerAborted")]
```

Comment:
21?

Reply:
Good spotting. I had a new log for 21, and I deleted it 😄